Sources/MapboxMaps/Foundation/Extensions/Core/MBXEdgeInsets.swift

```diff
@@ -9,11 +9,18 @@ internal extension EdgeInsets {
     }
 }
 
-internal extension UIEdgeInsets {
+extension UIEdgeInsets: Hashable {
```

Comment:
How does this interact if another library adds the same conformance but uses a different technique?

Reply:
Same question for CLLocationCoordinate2D and CGPoint